packages/block-library/src/embed/test/__snapshots__/index.js.snap

```diff
@@ -2,16 +2,8 @@
 
 exports[`core/embed block edit matches snapshot 1`] = `
 <div
-  class="components-placeholder wp-block-embed"
+  class="components-placeholder wp-block-embed is-large"
 >
-  <iframe
-    aria-hidden="true"
-    aria-label="resize-listener"
-    frameborder="0"
-    src="about:blank"
-    style="display:block;opacity:0;position:absolute;top:0;left:0;height:100%;width:100%;overflow:hidden;pointer-events:none;z-index:-1"
-    tabindex="-1"
-  />
```

Comment:
These snapshot changes do not seem like they are sensible, since they undo the fixes intended with #19825.

Reply:
Oh, I see that you updated the mocking in the tests, so that a value is provided. So maybe they are sensible. I'll take a closer look at those changes.